### PR TITLE
#21 Get rid of static methods

### PR DESCRIPTION
**Pull request by eparovyshnaya**
-  `TimeUtil` is refactored in *Orthodox OOP* style.
 - Tests are added

 - `BitmapUtil` is refactored accordingly. Untestable (no mock infrastruture, tightly cohesive with Android SDK).

### CODE REVIEW

File: app/src/main/java/ru/krivocraft/kbmp/core/utils/Milliseconds.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package ru.krivocraft.kbmp.core.utils;
```
Comment (ruspl-afed) [inline app/src/main/java/ru/krivocraft/kbmp/core/utils/Milliseconds.java:1]: Provide license header

File: app/src/main/java/ru/krivocraft/kbmp/core/utils/Seconds.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package ru.krivocraft.kbmp.core.utils;
```
Comment (ruspl-afed) [inline app/src/main/java/ru/krivocraft/kbmp/core/utils/Seconds.java:1]: Provide license header

File: app/src/main/java/ru/krivocraft/kbmp/core/utils/Seconds.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    public Seconds(int seconds) {
+        //todo: consider throwing IllegalArgumentException for negative values
+        this.seconds = seconds;
+    }
+
+    public String formatted() {
+        int secondsTail = seconds % 60;
+        int minutes = (seconds - secondsTail) / 60;
+
```
Comment (ruspl-afed) [inline app/src/main/java/ru/krivocraft/kbmp/core/utils/Seconds.java:16]: remove all empty lines, consider extracting sub-methods

File: app/src/main/java/ru/krivocraft/kbmp/fragments/LargePlayerFragment.java [hunk 4]
```diff
@@ -386,19 +387,19 @@ private void refreshUI() {
...
     private void updateStateShowers() {
-        int progress = TimeUtils.getSeconds(trackProgress);
-        int duration = TimeUtils.getSeconds(getTrackDuration());
+        int progress = new Milliseconds(trackProgress).seconds();
+        int duration = new Milliseconds(getTrackDuration()).seconds();
 
-        compositionProgressTextView.setText(TimeUtils.getFormattedTime(progress));
-        compositionDurationTextView.setText(TimeUtils.getFormattedTime((duration - progress) / 1000));
+        compositionProgressTextView.setText(new Seconds(progress).formatted());
+        compositionDurationTextView.setText(new Seconds((duration - progress) / 1000).formatted());
```
Comment (ruspl-afed) [inline app/src/main/java/ru/krivocraft/kbmp/fragments/LargePlayerFragment.java:402]: Do we need this conversion /1000 here?

File: app/src/test/java/ru/krivocraft/kbmp/utils/MillisecondsTest.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+package ru.krivocraft.kbmp.utils;
```
Comment (ruspl-afed) [inline app/src/test/java/ru/krivocraft/kbmp/utils/MillisecondsTest.java:1]: Add license headwer

File: app/src/test/java/ru/krivocraft/kbmp/utils/SecondsTest.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
+package ru.krivocraft.kbmp.utils;
```
Comment (ruspl-afed) [inline app/src/test/java/ru/krivocraft/kbmp/utils/SecondsTest.java:1]: add license header

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
Please address the feedback provided in comments

**Comment by ruspl-afed** [comment]
Regarding license headers there is a common issue #23  reported

File: app/src/main/java/ru/krivocraft/kbmp/fragments/LargePlayerFragment.java (hunk 4 above)
Comment (eparovyshnaya) [reply app/src/main/java/ru/krivocraft/kbmp/fragments/LargePlayerFragment.java:402]: Looks pretty much like an error.  
If `progress` is measured in seconds, then the second line looks suspicious. If in millis - the first one is erroneous and `Seconds` class is coded incorrectly. 

There should be a separate isssue for this. @zelenyhleb, please bake one.

File: app/src/main/java/ru/krivocraft/kbmp/core/utils/Seconds.java (hunk 3 above)
Comment (eparovyshnaya) [reply app/src/main/java/ru/krivocraft/kbmp/core/utils/Seconds.java:16]: Refuctoring was intended to be norrow and structural. Nevertheless *done* as  this particular one is small.

File: app/src/test/java/ru/krivocraft/kbmp/utils/SecondsTest.java (hunk 6 above)
Comment (eparovyshnaya) [reply app/src/test/java/ru/krivocraft/kbmp/utils/SecondsTest.java:1]: can we leave the whole license header work to issue #23 please?

**Pull request merged**